src/cloud/docker/docker-containers-service.md

```diff
@@ -0,0 +1,202 @@
+---
+group: cloud-guide
+title: Docker services containers
+functional_areas:
+  - Cloud
+  - Docker
+  - Configuration
+---
+
+
+The following containers provide the services required to build, deploy and run {{site.data.var.ee}} sites.
+
+{:.bs-callout-info}
+See the [service version values available]({{site.baseurl}}/cloud/docker/docker-containers.html#service-versions) for use when launching Docker.
+
+## Database container
+
+**Container name:** db<br/>
+**Docker base image:** [mariadb]<br/>
+**Ports exposed:** 3306<br/>
+
+The Database container is based on the [mariadb] image and includes the following volumes:
+
+-  `magento-db: /var/lib/mysql`
```

Comment:
I tried a number of different options.  Text block doesn't work because of the links in the description for Docker base image.  The current format is the best option for now, but mindful that it's not the best. Will revisit.